frontend/src/app/event/event-page/event-page.component.html

```diff
@@ -3,14 +3,24 @@
     <search-bar
       class="search-bar"
       [searchBarQuery]="searchBarQuery"
-      (searchBarQueryChange)="onSearchBarQueryChange($event)" />
+      (searchBarQueryChange)="this.searchUpdate.next($event)" />
   </div>
 
   <div class="container">
     <div class="events-grid">
       <div class="events-column">
         <!-- `eventsPerDay` is in the format of a list of tuples, where
                     eventGroup[0] is the date string and eventGroup[1] s a list of events -->
+        <div class="date-range">
+          <button mat-stroked-button (click)="showEvents(true)">
+            Previous Month
+          </button>
+          {{ startDate | date: 'shortDate' }} -
+          {{ endDate | date: 'shortDate' }}
```

Comment:
I would love to see a better representation up here, but this is non-blocking and can be fixed after deploy. Some custom logic may be needed for like "Today - Jan 26th, 2024" vs. "Jan 26th - Feb 26th, 2024"

Another issue to file for later, we should try to use the start minute/hour/second as first of the morning on start date and last of evening on end date, so that next pages always go to the next day.